### PR TITLE
[zh] Sync 1.11.8, 1.13.9 and 1.14.5 release notes into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync 1.11.8, 1.13.9 and 1.14.5 release notes into Chinese:
```
content\zh\news\releases\1.11.x\announcing-1.11.8\index.md
content\zh\news\releases\1.13.x\announcing-1.13.9\index.md
content\zh\news\releases\1.14.x\announcing-1.14.5\index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.14.x/announcing-1.14.5/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+---
+
+该版本包含了对 [CVE-2022-39278](/zh/news/security/istio-security-2022-007/#cve-2022-39278)
+的修复以及改进稳健性的漏洞修复。
+本发布说明描述了 Istio 1.14.4 和
+Istio 1.14.5 之间的不同之处。
+
+供参考，该版本包括（2022-10-04 发布的）Go 1.18.7
+中对 `archive/tar`、`net/http/httputil`
+和 `regexp` 包的安全修复。
```
Comment (windsonsea) [inline content/zh/news/releases/1.14.x/announcing-1.14.5/index.md:17]:
```suggestion
值得一提的是，该版本还包括（2022-10-04 发布的）Go 1.18.7 中对 `archive/tar`、
`net/http/httputil` 和 `regexp` 包的安全修复。
```

**Review by windsonsea — COMMENTED** [review]
不要过度折行，折行的目的是为了美观，现在是滥用折行了

**Comment by wilsonwu** [comment]
> 不要过度折行，折行的目的是为了美观，现在是滥用折行了

抱歉，由于我是分屏编辑，所以按照分屏宽度折行，我已经了解规则，40个中文左右，后续提交会遵循相关规则。

File: content/zh/news/releases/1.14.x/announcing-1.14.5/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/news/releases/1.14.x/announcing-1.14.5/index.md]: All fixed in new commit.

**Review by windsonsea — APPROVED** [review]

**Pull request merged**